Resources/Prototypes/_NF/Body/Organs/goblin_organs.yml

```diff
@@ -0,0 +1,200 @@
+- type: entity
+  parent: OrganHumanBrain
+  id: OrganGoblinBrain
+  name: brain
+  suffix: Goblin
+  description: "The source of your woes."
+  components:
+  - type: Sprite
+    sprite: _NF/Mobs/Species/Goblin/organs.rsi
+    state: brain
+  - type: Organ
+  - type: Input
+    context: "ghost"
+  - type: Brain
+  - type: InputMover
+  - type: Examiner
+  - type: BlockMovement
+  - type: BadFood
+  - type: Tag
+    tags:
+      - Meat
+
+- type: entity
+  parent: OrganHumanEyes
+  id: OrganGoblinEyes
+  name: eyes
+  suffix: Goblin
+  components:
+  - type: Sprite
+    sprite: _NF/Mobs/Species/Goblin/organs.rsi
+    layers:
+      - state: eyeball-l
+      - state: eyeball-r
+
+- type: entity
+  parent: OrganHumanTongue
+  id: OrganGoblinTongue
+  name: tongue
+  suffix: Goblin
+  components:
+  - type: Sprite
+    sprite: _NF/Mobs/Species/Goblin/organs.rsi
+    state: tongue
+
+- type: entity
+  parent: OrganHumanAppendix
+  id: OrganGoblinAppendix
+  name: appendix
+  suffix: Goblin
+  components:
+  - type: Sprite
+    sprite: _NF/Mobs/Species/Goblin/organs.rsi
+    layers:
+    - state: appendix
+    - state: appendix-inflamed
+      visible: false
+
+- type: entity
+  parent: OrganHumanEars
+  id: OrganGoblinEars
+  name: ears
+  suffix: Goblin
+  components:
+  - type: Sprite
+    sprite: _NF/Mobs/Species/Goblin/organs.rsi
+    state: ears
+
+- type: entity
+  parent: BaseHumanOrgan
+  id: OrganGoblinLungs
+  name: lungs
+  suffix: Goblin
+  description: "They breathe business."
+  components:
+  - type: Sprite
+    sprite: _NF/Mobs/Species/Goblin/organs.rsi
+    layers:
+      - state: lung-l
+      - state: lung-r
+  - type: Lung
+  - type: Metabolizer
+    removeEmpty: true
+    solutionOnBody: false
+    solution: "Lung"
+    metabolizerTypes: [ Animal ]
+    groups:
+    - id: Gas
+      rateModifier: 100.0
+  - type: SolutionContainerManager
+    solutions:
+      organ:
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 10
+      Lung:
+        maxVol: 100.0
+        canReact: false
+      food:
+        maxVol: 5
+        reagents:
+        - ReagentId: UncookedAnimalProteins
+          Quantity: 5
+
+- type: entity
+  parent: OrganHumanHeart
+  id: OrganGoblinHeart
+  name: heart
+  suffix: Goblin
+  components:
+  - type: Sprite
+    sprite: _NF/Mobs/Species/Goblin/organs.rsi
+    state: heart-on
+  - type: Metabolizer
+    metabolizerTypes: [ Goblin ]
+
+- type: entity
+  parent: [OrganAnimalStomach, OrganHumanStomach]
+  id: OrganGoblinStomach
+  name: stomach
+  suffix: Goblin
+  description: "Shrooms go in here."
+  noSpawn: true
+  components:
+  - type: Sprite
+    sprite: _NF/Mobs/Species/Goblin/organs.rsi
+    state: stomach
+  - type: SolutionContainerManager
+    solutions:
+      stomach:
+        maxVol: 50
+      food:
+        maxVol: 5
+        reagents:
+        - ReagentId: UncookedAnimalProteins
+          Quantity: 5
+  - type: Stomach
+    reverseFoodQuality: true
```

Comment:
Need to remove this to the food system PR.
Just uncomment it here and we will restore it later.